### DISCUSSION TITLE
Add the ability to sort on relations TABLE layout

### DISCUSSION THREAD

**Discussion by NigmaX** [directus/directus#16125] [Feature Requests]
Currently, when creating any relation interface with a sort field.
The "sorting" ability will be applied only on the list view but not on the new table layout view.

Is there any plan to apply the sorting ability to the table layout?
Thanks. :)

**Comment by azrikahar** [comment]
This was raised out for O2M interface table layout in #15732 with an ongoing PR #15733 👍 That being said, the same support should also probably be added to M2M interface for better consistency.

**Reply by NigmaX** [reply]
@azrikahar How can i help pushing it forward?
Should i work on the branch in the PR? Or open a new PR?

**Reply by azrikahar** [reply]
It'd be helpful if you can check it out locally to verify the fix and possibly trying to see are there any uncovered edge cases!

At first glance it seems to be based on existing codes such as:

https://github.com/directus/directus/blob/228a5ca15f981f01c7072b641d53e1edf562629a/packages/shared/src/composables/use-items.ts#L196-L207

https://github.com/directus/directus/blob/228a5ca15f981f01c7072b641d53e1edf562629a/app/src/interfaces/list-o2m/list-o2m.vue#L412-L421

so it doesn't seem to be too big of a lift, but verifying it would still be for the best. I'm not sure whether you can work on that branch (maybe apart from making the PR to their fork so that they can merge it into their PR), but I think once the o2m solution is confirmed, we'll be able to replicate it over in m2m afterwards.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by jvgadea** [comment]
I ran into the same problem. When you put it as a table layout, it doesn't allow you to sort it. And if the list has several pages either. Is there a solution for this problem? It is quite difficult at the user level to work like this
